lib/screens_web/controllers/v2/screen_api_controller.ex

```diff
@@ -90,8 +89,7 @@ defmodule ScreensWeb.V2.ScreenApiController do
             update_visible_alerts?: true,
             logging_options: %{
```

Comment:
This whole option (`logging_options`) was only used on Triptychs, so we should be able to completely remove it. For that matter, I think it was the only reason options needed to be passed down from `ScreenData` into candidate generators at all.

Reply:
Nice catch. I didn't notice that.